include/graft_manager.h

```diff
@@ -331,6 +328,8 @@ class GraftServer final
     constexpr static std::pair<const char *, int> m_methods[] = {
         _M(GET), _M(POST), _M(PUT), _M(DELETE), _M(HEAD) //, _M(CONNECT)
     };
+public:
+    std::atomic_bool ready;
```

Comment:
as it not supposed to be *set* outside, please make it public `bool ready() const`  method instead

Reply:
done. thanks